Automattic/woocommerce-payments
# Revert "Adding manual option to deposit schedule selection (#4715)"

This reverts commit de82c8f40181b33e961d30c508a80b23b8d7423f (PR #4715, original issue #4696)

#### Changes proposed in this Pull Request

This reverts the changes introduced in PR #4715 (Adding manual option to deposit schedule selection). We are reverting these changes as we have decided not to ship the Manual Deposits project (pdjTHR-1DS-p2) in WCPay 5.0.

| Before | After |
| --- | --- |
| <img width="694" alt="Screen Shot 2022-11-04 at 3 01 29 pm" src="https://user-images.githubusercontent.com/1527164/199882985-2e110ef5-ae50-4d61-a7c7-73c707b2c0c0.png"> | <img width="692" alt="Screen Shot 2022-11-04 at 3 00 16 pm" src="https://user-images.githubusercontent.com/1527164/199883011-c0c974e7-0fed-49e5-ad03-e855b5747306.png"> |

#### Testing instructions

* Navigate to **Payments → Settings**.
* Scroll down to the "Deposits" section.
* Observe there is no "Manual" radio button for the "Deposit schedule" field.
* Switching between the existing automatic schedule options (`Daily`, `Weekly` and `Monthly`) continues to work as expected.

<img width="1075" alt="Screen Shot 2022-11-04 at 2 55 31 pm" src="https://user-images.githubusercontent.com/1527164/199882717-ed02a20e-5b15-4d73-be3d-2f056c6d0cae.png">

-------------------


**Post merge**
